### PR TITLE
Namespace: lyrics_bow

### PR DESCRIPTION
**Pull request by bmcfee**
Implements #81

---
This change is [Reviewable](https://reviewable.io/reviews/marl/jams/99)

### CODE REVIEW

**Comment by bmcfee** [comment]
I think this one's ready for CR; @ejhumphrey?

**Comment by bmcfee** [comment]
:dash: anyone?  @justinsalamon ?

**Comment by justinsalamon** [comment]
sorry no cycles right now, maybe in a month (seriously)

**Comment by bmcfee** [comment]
> sorry no cycles right now, maybe in a month (seriously)

months

**Comment by justinsalamon** [comment]
I did say "maybe" :P

Seriously though, no cycles right now, sorry.

**Comment by bmcfee** [comment]
This is how open source dies. :cry:

File: docs/namespaces/lyrics.rst [hunk 1]
```diff
@@ -29,3 +29,29 @@ The required ``value`` field can contain arbitrary text data, e.g., lyrics.
 .. note::
     ``confidence`` is an unconstrained field, and may contain arbitrary data.
 
+lyrics_bow
+~~~~~~~~~~
+Time-aligned bag-of-words or bag-of-ngrams.
+
+    ===== ======== ====== ==========
+    time  duration value  confidence
+    ===== ======== ====== ==========
```
Comment (stefan-balke) [inline docs/namespaces/lyrics.rst]: I did a quick check and it looks good to me.

Comment (stefan-balke) [inline docs/namespaces/lyrics.rst]: !fixed

Comment (stefan-balke) [inline docs/namespaces/lyrics.rst]: Sorry for the spam. Maybe this project should move to reviewable too?
Thought that it was way more intuitive :)

**Comment by bmcfee** [comment]
Thanks @stefan-balke!

**Pull request merged**